### PR TITLE
Add release notes for Hive INSERT and DELETE

### PR DESCRIPTION
**Pull request by dain**

### CODE REVIEW

**Comment by haozhun** [comment]
Temporarily, partition columns specified in `DELETE` must be `VARCHAR` type. But create table can take any type.

File: presto-docs/src/main/sphinx/release/release-0.123.rst [hunk 1]
```diff
@@ -19,3 +19,23 @@ creating a Hive table you can specify the file format.  To list all available ta
 properties, run the following query::
 
     SELECT * FROM system.metadata.table_properties
+
+Hive Changes
+------------
+
+We have implemented ``INSERT`` and ``DELETE`` for Hive.  Both ``INSERT`` and ``CREATE``
+statements support partitioned tables.  For example, to create a partitioned table
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.123.rst]: Comma after "table

File: presto-docs/src/main/sphinx/release/release-0.123.rst [hunk 2]
```diff
@@ -19,3 +19,23 @@ creating a Hive table you can specify the file format.  To list all available ta
...
+execute the following::
+
+    CREATE TABLE orders (
+       order_date DATE,
+       order_region VARCHAR,
+       order_id BIGINT,
+       order_info VARCHAR
+    ) WITH (partitioned_by = ARRAY[ 'order_date', 'order_region' ])
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.123.rst]: Remove spaces around brackets
`ARRAY['order_date', 'order_region']`

File: presto-docs/src/main/sphinx/release/release-0.123.rst [hunk 3]
```diff
@@ -19,3 +19,23 @@ creating a Hive table you can specify the file format.  To list all available ta
...
+
+To ``DELETE`` from a Hive table, you must specify a ``WHERE`` clause that matches
+entire partitions.  For example, to delete from the above table execute the following::
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.123.rst]: Comma after "table"

File: presto-docs/src/main/sphinx/release/release-0.123.rst [hunk 4]
```diff
@@ -19,3 +19,23 @@ creating a Hive table you can specify the file format.  To list all available ta
...
+
+    DELETE FROM orders
+    WHERE order_date = DATE '2015-10-15' and order_region = 'APAC';
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.123.rst]: Capitalize "AND"

Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.123.rst]: Remove semicolon at end

**Pull request closed without merging**